### PR TITLE
Added the ability to clear the log via a REST call, plus some other small features

### PR DESCRIPTION
**Pull request by alonweiss-sl**
- Quit AnyProxy using ctrl+c
- Added another sample rule to return 200 OK for all requests

### CODE REVIEW

**Comment by ottomao** [comment]
Hi,
Thanks for your contribution.
Why would you want to clea logs by rest call ? In my opinion, logs should not be cleared in db. Users who want to analyze logs could filter the logs by their own.

What's more, the modification of package.json should be kept in your own branch, instead of merging to this repo.

**Comment by alonweiss-sl** [comment]
Hi,
I'm using anyproxy as a part of a component test,  where in every test the
component is expected to make several requests to the server (which is
mocked by a rule). At the end of each test I grab the log from anyproxy,
inspect it with assertions,  then clean it for the next test. This is
therefore a requirement for me in order to use it. The only alternative is
to host anyproxy in the same process of the test (mocha in my case),  but i
won't be able to do this with other frameworks I'll soon need (Java, .
Net,  etc.)

Changing package.json is not meant to be merged,  it's just something i had
to do so I can publish it and install it on other machines using npm,
until this merge is approved (or not..).  I changed it after making the
pull request so it doesn't get merged to the main project.

This is actually my first pull request ever,  so I might have made some
mistakes along the way.

What so you think?

-Alon.

**Comment by ottomao** [comment]
Hi,
By using your rest call, we can clear all the logs without restart proxy server. But this seems to be a special approach to meet your demands, not a basic function for AnyProxy. 

For those who want to analyze web status, we plan to develop a request-monitor service. With this general service, third-party applications will be able to detect/save/anaylyze realtime information of all the requests. This feature will be implemented by websocket-interface, and here is a prototype for browser : https://github.com/alibaba/anyproxy/blob/master/web/lib/anyproxy_wsUtil.js . More documents and sample usages are on still on their way.

In short, I think a request-monitor is more suitable for AnyProxy to meet all kinds of situation, not just a rest-to-clear service.
